src/Flex.php

```diff
@@ -39,13 +37,41 @@
  */
 class Flex implements PluginInterface, EventSubscriberInterface
 {
+    /**
+     * @var Composer
+     */
     private $composer;
+
+    /**
+     * @var IOInterface
+     */
     private $io;
+
+    /**
+     * @var Options
+     */
     private $options;
+
+    /**
+     * @var Configurator
+     */
     private $configurator;
+
+    /**
+     * @var Downloader
+     */
     private $downloader;
+
+    /**
+     * @var array
+     */
     private $postInstallOutput = [''];
+
+    /**
+     * @var \Closure
+     */
```

Comment:
Technically they should be `Some|null` also.. but i agree with @chalasr as well on not documenting private stuff.

in general :+1: for inlined :)